packages/terra-application-links/src/tabs/ApplicationTabs.scss

```diff
@@ -77,6 +82,10 @@
     }
   }
 
+  .tab:hover {
+    color: var(--terra-application-tabs-hover-color, #fff) !important;
```

Comment:
Do we want to hover as white or clear? I think this was something Neil noticed with buttons